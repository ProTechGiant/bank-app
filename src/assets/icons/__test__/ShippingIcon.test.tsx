import { create } from "react-test-renderer";

import { ShippingIcon } from "../ShippingIcon";

describe("icon/ShippingIcon", () => {
  it("renders a ShippingIcon", () => {
    const result = create(<ShippingIcon />);

    expect(result).toMatchSnapshot();
  });
});
