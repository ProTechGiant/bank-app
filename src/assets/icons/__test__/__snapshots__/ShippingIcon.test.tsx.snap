// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/ShippingIcon renders a ShippingIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="80"
  bbWidth="80"
  fill="none"
  focusable={false}
  height={80}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 80,
        "width": 80,
      },
    ]
  }
  vbHeight={80}
  vbWidth={80}
  width={80}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGRect
      fill={
        {
          "payload": 4294134835,
          "type": 0,
        }
      }
      height="80"
      propList={
        [
          "fill",
        ]
      }
      rx="40"
      width="80"
      x="0"
      y="0"
    />
    <RNSVGPath
      d="M55.8636 32.25H49.7273V24H21.0909C18.8409 24 17 25.8562 17 28.125V50.8125H21.0909C21.0909 54.2362 23.8318 57 27.2273 57C30.6227 57 33.3636 54.2362 33.3636 50.8125H45.6364C45.6364 54.2362 48.3773 57 51.7727 57C55.1682 57 57.9091 54.2362 57.9091 50.8125H62V40.5L55.8636 32.25ZM27.2273 53.9062C25.5295 53.9062 24.1591 52.5244 24.1591 50.8125C24.1591 49.1006 25.5295 47.7188 27.2273 47.7188C28.925 47.7188 30.2955 49.1006 30.2955 50.8125C30.2955 52.5244 28.925 53.9062 27.2273 53.9062ZM54.8409 35.3438L58.85 40.5H49.7273V35.3438H54.8409ZM51.7727 53.9062C50.075 53.9062 48.7045 52.5244 48.7045 50.8125C48.7045 49.1006 50.075 47.7188 51.7727 47.7188C53.4705 47.7188 54.8409 49.1006 54.8409 50.8125C54.8409 52.5244 53.4705 53.9062 51.7727 53.9062Z"
      fill={
        {
          "payload": 4294638330,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
